### PR TITLE
fix sorting columns in frids

### PR DESCRIPTION
**Pull request by ct-goranpalibrk**
Signed-off-by: CT\pgoran <goran.palibrk@comtrade.com>

### CODE REVIEW

**Comment by ct-goranpalibrk** [comment]
Hi,

This PR solves issues #1599  and #1623

**Comment by LeoNerdoG** [comment]
I have verified that this PR successfully resolves issue #1599 and #1623. It can be merged when all tests complete.

File: console/module/account/src/main/java/org/eclipse/kapua/app/console/module/account/shared/util/GwtKapuaAccountModelConverter.java [hunk 1]
```diff
@@ -52,6 +52,9 @@ public static AccountQuery convertAccountQuery(PagingLoadConfig loadConfig, GwtA
         if (sortField.equals("modifiedOnFormatted")) {
             sortField = AccountPredicates.MODIFIED_ON;
         }
+        if (sortField.equals("modifiedByName")) {
```
Comment (muros-ct) [inline console/module/account/src/main/java/org/eclipse/kapua/app/console/module/account/shared/util/GwtKapuaAccountModelConverter.java]: use else if

File: console/module/authentication/src/main/java/org/eclipse/kapua/app/console/module/authentication/shared/util/GwtKapuaAuthenticationModelConverter.java [hunk 2]
```diff
@@ -73,6 +73,9 @@ public static CredentialQuery convertCredentialQuery(PagingLoadConfig loadConfig
             andPredicate.and(new AttributePredicateImpl<CredentialType>(CredentialPredicates.CREDENTIAL_TYPE, convertCredentialType(gwtCredentialQuery.getType()), Operator.EQUAL));
         }
         String sortField = StringUtils.isEmpty(loadConfig.getSortField()) ? CredentialPredicates.CREDENTIAL_TYPE : loadConfig.getSortField();
+        if (sortField.equals("expirationDateFormatted")) {
```
Comment (muros-ct) [inline console/module/authentication/src/main/java/org/eclipse/kapua/app/console/module/authentication/shared/util/GwtKapuaAuthenticationModelConverter.java]: Use if / else if approach same as everywhere else.

File: console/module/authorization/src/main/java/org/eclipse/kapua/app/console/module/authorization/server/GwtAccessPermissionServiceImpl.java [hunk 3]
```diff
@@ -132,6 +132,9 @@ public void delete(GwtXSRFToken gwtXsrfToken, String scopeShortId, String access
                     accessPermissionQuery.setLimit(loadConfig.getLimit());
                     accessPermissionQuery.setOffset(loadConfig.getOffset());
                     String sortField = StringUtils.isEmpty(loadConfig.getSortField()) ? "createdOn" : loadConfig.getSortField();
+                    if (sortField.equals("createdOnFormatted")) {
```
Comment (muros-ct) [inline console/module/authorization/src/main/java/org/eclipse/kapua/app/console/module/authorization/server/GwtAccessPermissionServiceImpl.java]: Again same approach not ternary and if.

File: console/module/authorization/src/main/java/org/eclipse/kapua/app/console/module/authorization/shared/util/GwtKapuaAuthorizationModelConverter.java [hunk 4]
```diff
@@ -122,8 +122,11 @@ public static RoleQuery convertRoleQuery(PagingLoadConfig loadConfig, GwtRoleQue
             roleQuery.setPredicate(new AttributePredicateImpl<String>(RolePredicates.NAME, gwtRoleQuery.getName(), Operator.LIKE));
         }
         String sortField = StringUtils.isEmpty(loadConfig.getSortField()) ? RolePredicates.NAME : loadConfig.getSortField();
-        if (sortField.equals("createdOnFormatted")) {
-            sortField = RolePredicates.CREATED_ON;
+        if (sortField.equals("modifiedOnFormatted")) {
+            sortField = RolePredicates.MODIFIED_ON;
+        }
```
Comment (muros-ct) [inline console/module/authorization/src/main/java/org/eclipse/kapua/app/console/module/authorization/shared/util/GwtKapuaAuthorizationModelConverter.java]: else if

File: console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/shared/util/GwtKapuaJobModelConverter.java [hunk 5]
```diff
@@ -186,6 +186,12 @@ public static JobStepQuery convertJobStepQuery(GwtJobStepQuery gwtJobStepQuery,
             sortCriteria = new FieldSortCriteria(sortField, sortOrder);
         } else {
             String sortField = StringUtils.isEmpty(loadConfig.getSortField()) ? JobStepPredicates.STEP_INDEX : loadConfig.getSortField();
+            if (sortField.equals("jobStepName")) {
+                sortField = JobStepPredicates.JOB_STEP_NAME;
+            }
```
Comment (muros-ct) [inline console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/shared/util/GwtKapuaJobModelConverter.java]: else if

File: console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/shared/util/GwtKapuaJobModelConverter.java [hunk 6]
```diff
@@ -242,6 +248,15 @@ public static TriggerQuery convertTriggerQuery(GwtTriggerQuery gwtTriggerQuery,
                 .and(kapuaPropertyTypeAttributePredicate);
 
         String sortField = StringUtils.isEmpty(loadConfig.getSortField()) ? TriggerPredicates.ENTITY_ID : loadConfig.getSortField();
+        if (sortField.equals("triggerName")) {
+            sortField = TriggerPredicates.TRIGGER_NAME;
+        }
```
Comment (muros-ct) [inline console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/shared/util/GwtKapuaJobModelConverter.java]: all with else if

File: console/module/user/src/main/java/org/eclipse/kapua/app/console/module/user/shared/util/GwtKapuaUserModelConverter.java [hunk 7]
```diff
@@ -67,6 +67,12 @@ public static UserQuery convertUserQuery(PagingLoadConfig loadConfig, GwtUserQue
         if (sortField.equals("username")) {
             sortField = UserPredicates.NAME;
         }
+        if (sortField.equals("modifiedByName")) {
+            sortField = UserPredicates.MODIFIED_BY;
+        }
```
Comment (muros-ct) [inline console/module/user/src/main/java/org/eclipse/kapua/app/console/module/user/shared/util/GwtKapuaUserModelConverter.java]: else if

**Comment by ct-goranpalibrk** [comment]
I made requested PR changes.

**Pull request merged**